src/running-policies.md

```diff
@@ -0,0 +1,230 @@
+# Running Policies
+
+Running policies is by far the most common operation that a Kubernetes
+operator will perform. You can configure as many policies as you want,
```

Comment:
I agree with @cmurphy .

I would write something like:
> Define policies is by far the most common operation that a Kubernetes administrator will perform.
> You can declare as many policies....

I would also use the terms `validate`, `evaluate` or, maybe better, `enforce` instead of `run`.